src/main/java/com/amazonaws/services/kinesis/multilang/MultiLangProtocol.java

```diff
@@ -100,6 +101,18 @@ boolean shutdown(IRecordProcessorCheckpointer checkpointer, ShutdownReason reaso
     }
 
     /**
+     * Writes a {@link ShutdownRequestedMessage} to the child process's STDIN and waits for the child process to respond with a
+     * {@link StatusMessage} on its STDOUT.
+     *
+     * @param checkpointer A checkpointer.
+     * @return Whether or not this operation succeeded.
+     */
+    boolean shutdownRequested(IRecordProcessorCheckpointer checkpointer) {
```

Comment:
Could you please add some tests for this method in [MultiLangProtocolTest.java](https://github.com/awslabs/amazon-kinesis-client/blob/master/src/test/java/com/amazonaws/services/kinesis/multilang/MultiLangProtocolTest.java)